api/client/lib.py

```diff
@@ -184,6 +186,11 @@ def get_data(url, headers, params=None, logger=None):
     """
     base_log_record = dict(route=url, params=params)
     retry_count = 0
+
+    # append version info
+    headers['api_client_version'] = __version__
```

Comment:
Also unfortunately batch_client has its own version of `get_data()` since it's using a different HTTP library (`tornado` instead of `requests`). So, need to duplicate these headers over there too.

https://github.com/gro-intelligence/api-client/blob/f0003b78e44de2dd0c5d70595f7b30da303182dc/api/client/batch_client.py#L32

Reply:
done